tests/unit/client/test_mixin_client.py

```diff
@@ -537,3 +542,12 @@ def raise_error():
     with pytest.raises(RubiconException) as e:
         experiment.tags()
     assert "all configured storage backends failed" in str(e)
+
+
+def test_add_comments(project_client):
```

Comment:
can we add client level tests for removing comments and the error message as well? like the last two tag tests here:
> https://github.com/capitalone/rubicon-ml/blob/main/tests/unit/client/test_mixin_client.py#L519